### PR TITLE
feat(OCI): document how to deploy Pro Kubernetes cluster on OpenShift

### PR DESCRIPTION
**Pull request by SamirPS**
Document how to deploy an Ubuntu Pro Kubernetes cluster on OpenShift.

### CODE REVIEW

**Comment by SamirPS** [comment]
Preview here: https://canonical-public-cloud--33.com.readthedocs.build/en/33/oci/oci-how-to/deploy-ubuntu-pro-cluster-on-openshift/

File: oci/oci-how-to/deploy-ubuntu-pro-cluster-on-openshift.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+	"RHCOS is the only supported operating system for OpenShift Container Platform control plane, or master, machines. While RHCOS is the default operating system for all cluster machines, you can create compute machines, which are also known as worker machines, that use RHEL as their operating system."
+
+	-- `OpenShift docs <https://docs.openshift.com/container-platform/4.8/architecture/architecture-rhcos.html#rhcos-about_architecture-rhcos>`_
+
+
+Conclusion
+~~~~~~~~~~
+
+You can reach out to `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_ for attaching the cluster nodes to an Ubuntu Pro subscription, or contact rocks@canonical.com if you need additional support.
```
Comment (k-dimple) [inline oci/oci-how-to/deploy-ubuntu-pro-cluster-on-openshift.rst:23]: Typo: Remove the 'to'  "You can reach out here for .."

**Review by k-dimple — CHANGES_REQUESTED** [review]

File: oci/oci-how-to/deploy-ubuntu-pro-cluster-on-openshift.rst (hunk 1 above)
Comment (SamirPS) [reply oci/oci-how-to/deploy-ubuntu-pro-cluster-on-openshift.rst]: Done

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
this is not the intention.

We've agreed that OpenShift, Nutanix and Tanzu can all be seen as generic infrastructures. Furthermore, this is the OCI section and not the "Clouds" one, so it doesn't make sense to have an infrastructure-related guide in here.

The goal is: "How to deploy an Ubuntu Pro container on OpenShift, Tanzu, Nutanix". This guide should have an introduction explaining the suggested infrastructures, how they are generics, what their limitations are, and how/who to contact Canonical for turning these infrastructures into Pro infrastructures. But the goal will always be "deploy an ubuntu pro container", cause this is an OCI guide, and not a cloud one.

**Comment by SamirPS** [comment]
Closing this per https://github.com/canonical/ubuntu-cloud-docs/pull/32#discussion_r1169025577.

**Pull request closed without merging**